src/components/SettingsList/SettingsList.tsx

```diff
@@ -87,13 +87,15 @@ const SettingsList = () => {
               </label>
             </div>
           </div>
-          <div className='settings-item'>
+          <div className={`settings-item ${unsavedSettings.playerAccess.canModifyAllTurns ? 'settings-item-disabled' : ''}`}
+            title="Only applies if the player permission 'Owner Only' is enabled.">
             <p>Allow players to set their turns if player owned.</p>
             <div className='settings-item-input'>
-              <label className='switch'>
+              <label className={`switch ${unsavedSettings.playerAccess.canModifyAllTurns ? 'disabled' : ''}`}>
```

Comment:
Same things as above
```suggestion
              <label className={clsx('switch', { disabled: unsavedSettings.playerAccess.canModifyAllTurns })}>
```